scarb/src/flock.rs

```diff
@@ -149,6 +149,7 @@ pub struct AdvisoryLock<'f> {
 }
 
 #[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
Why?

Reply:
```
warning: field `0` is never read
   --> scarb/src/flock.rs:152:30
    |
152 | pub struct AdvisoryLockGuard(Arc<FileLockGuard>);
    |            ----------------- ^^^^^^^^^^^^^^^^^^
    |            |
    |            field in this struct
    |
    = note: `AdvisoryLockGuard` has a derived impl for the trait `Debug`, but this is intentionally ignored during dead code analysis
    = note: `#[warn(dead_code)]` on by default
help: consider changing the field to be of unit type to suppress this warning while preserving the field numbering, or remove the field
    |
152 | pub struct AdvisoryLockGuard(());
    |                              ~~

```

Reply:
let's drop `Debug` impl then perhaps?

Reply:
Sure! If that's the only reason for this :D

Reply:
```suggestion
```